### PR TITLE
fix typos in html documentations

### PR DESCRIPTION
**Pull request by garanews**
fix typos in html documentations

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
@garanews, Looks good.

File: content/html/concepts/elements/terms/noscript/noscript.md [hunk 1]
```diff
@@ -38,6 +38,6 @@ In the example below, if JavaScript isn't enabled, the [`<script>`](https://www.
 </noscript>
 ```
 
-The output might look like this if Javascipt is disabed or not avalible:
+The output might look like this if Javascript is disabled or not available:
```
Comment (Dusch4593) [inline content/html/concepts/elements/terms/noscript/noscript.md:41]:
```suggestion
The output might look like this if JavaScript is disabled or not available:
```

File: content/html/concepts/videos/videos.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ It is generally discouraged to use videos as the background for a webpage. In ma
 - If the video is purely decorative, use `aria-hidden="true"` to denote to screen readers that the video provides no additional context or content and is purely decorative
 - Use the `prefers-reduced-motion` CSS media query in order to disable autoplaying videos for certain users
```
Comment (Dusch4593) [inline content/html/concepts/videos/videos.md:69]:
```suggestion
- If the video is purely decorative, `aria-hidden="true"` can denote to screen readers that the video provides no additional context or content and is purely decorative.
- The `prefers-reduced-motion` CSS media query can be used to disable autoplaying videos for certain users.
```

File: content/html/concepts/elements/terms/u/u.md [hunk 3]
```diff
@@ -30,5 +30,5 @@ The `<u>` element should only contain text content or other tags meant for text
 ## Example
 
 ```html
-<p>You can use this tag to indicate things such as <u>mispelt</u> words.</p>
+<p>You can use this tag to indicate things such as <u>misspelt</u> words.</p>
```
Comment (Dusch4593) [inline content/html/concepts/elements/terms/u/u.md:33]:
```suggestion
<p>You can use this element to indicate things such as <u>misspelt</u> words.</p>
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@garanews Finished with my review. Aside from the comments I posted, here are other suggestions for this PR:

Under `noscript.md`:
- On line # 15, let's change "Javascript" to "JavaScript".
- On line # 19, under the "Syntax" section, let's change the code block signature from \```html to \```pseudo.

Under `u.md`:
- On line # 18, let's add a `>` in front of `**Note:**`.
- - On line # 22, under the "Syntax" section, let's change the code block signature from \```html to \```pseudo.

Under `videos.md`:
- On line # 3, let's shorten the `Description` to (ideally) 150 characters or less.
- On line # 15, let's **bold** "videos" like so: `Embedding **videos** within a site ...`.
- Let's add periods (`.`) to the end of the list items on lines # 47 through 50 as well as lines # 75 through # 78.

**Comment by garanews** [comment]
just missing the downsizing of description, I would't remove something relevant...

**Comment by Dusch4593** [comment]
@garanews Agreed, relevance shouldn't be left out. The `Description` (usually on line # 3 of any entry file) is more for SEO-purposes than anything else. For the one in `video.md`, maybe something like the following:

```
Embedding videos creates more lively and informative web pages that are more attractive to visitors but costly in bandwidth and accessibility.
```

**Review by Dusch4593 — APPROVED** [review]
@garanews LGTM for merge! 🚀 

I made a few small edits in 491f4ea7c57870834498d9c4677faeee57277610.

**Pull request merged**